utils/debug.c

```diff
@@ -147,6 +171,24 @@ void setup_color(enum color_setting color, char *pager)
 	}
 }
 
+static const char *get_color(char code)
+{
+	unsigned i;
+
+	for (i = 0; i < ARRAY_SIZE(colors); i++) {
+		if (code == colors[i].code) {
+			if (out_color == COLOR_ON) {
+			        if (format_mode == FORMAT_HTML)
+				        return colors[i].html_color;
+                                else
+				        return colors[i].color;
+                        }
```

Comment:
Indentation was broken.  Also I prefer having fewer blocks in a loop.  Please consider using 'continue' more frequently.